cmd/docker/docker.go

```diff
@@ -57,6 +57,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags = cmd.Flags()
 	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
 	flags.StringVar(&opts.ConfigDir, "config", cliconfig.Dir(), "Location of client config files")
+	flags.BoolVarP(&opts.TLSGetPass, "tlsgetpass", "", cliconfig.TLSGetPass(), "Request passphrase if needed to decrypt TLS keys")
```

Comment:
I agree with @cyli. If we're replacing what was an error with more intelligent behaviour, I think that's a great improvement that almost certainly doesn't need an exposed flag, even if it's new behaviour.